.circleci/config.yml

```diff
@@ -62,10 +62,25 @@ jobs:
           keys:
             - stripe-sdk-cache-{{ checksum "stripe_version.txt" }}
 
+      - run:
+          name: Store Xcode Version
+          command: xcodebuild -version > xcode_version.txt
+
+      - restore_cache:
+          name: Restore Carthage Cache
+          keys:
+            - carthage-cache-{{ checksum "Cartfile.resolved" }}-{{ checksum "xcode_version.txt" }}
```

Comment:
We combine the checksum of `Cartfile.resolved` with the current Xcode version so that we know it will rebuild dependencies if we use a new version of Xcode on CI.